### PR TITLE
beta connections-2.0

### LINKED ISSUE #14000: Beta Connectors 2.0
Switch the connectors-2.0 feature to beta.

### PR DESCRIPTION
**Pull request by KyleAure**
fixes #14000

### CODE REVIEW

**Comment by KyleAure** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/connectors-2.0/io.openliberty.connectors-2.0.feature [hunk 1]
```diff
@@ -12,5 +12,5 @@ Subsystem-Category: JakartaEE9Application
  com.ibm.websphere.appserver.eeCompatible-9.0
 -bundles=com.ibm.ws.app.manager.rar, \
  com.ibm.ws.jca.1.7.jakarta
-kind=noship
-edition=full
+kind=beta
+edition=core
```
Comment (aguibert) [inline dev/com.ibm.websphere.appserver.features/visibility/public/connectors-2.0/io.openliberty.connectors-2.0.feature:16]: JCA should be `edition=base` for as much as many features as we can. If there are features in `core` that require the JCA 2.0 API then we can leave the API feature in `core`, but aside from that we should move the features to `base`

**Review by aguibert — APPROVED** [review]

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
#libbybot

**Pull request merged**